5.5/run-mysqld.sh

```diff
@@ -94,6 +103,29 @@ if [ "$1" = "mysqld" -a ! -d "$MYSQL_DATADIR/mysql" ]; then
 		EOSQL
 	fi
 	mysqladmin $admin_flags flush-privileges shutdown
+}
+
+function generate_config() {
+	# Time zone.
```

Comment:
the fact it is not in config file does not mean it has no default ... check
mysql docs
On Mar 24, 2015 2:51 PM, "Martin Nagy" <notifications@github.com> wrote:

> In 5.5/run-mysqld.sh
> <https://github.com/openshift/mysql/pull/25#discussion_r27028488>:
>
> > @@ -94,6 +103,29 @@ if [ "$1" = "mysqld" -a ! -d "$MYSQL_DATADIR/mysql" ]; then
> >  		EOSQL
> >  	fi
> >  	mysqladmin $admin_flags flush-privileges shutdown
> > +}
> > +
> > +function generate_config() {
> > +	# Time zone.
>
> There is no such thing. The default is that it's not in the config file at
> all. I could use another variable with another name, but that seems ugly
> and unnecessary (one more variable to keep track of..)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/mysql/pull/25/files#r27028488>.
>